scania-digital-design-system/sdds
# fix(message): added theme to controls for message component

**Describe pull-request**  
Reverted back to add theme as part of controls

**Solving issue**  
Fixes: -

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in components -> message
3. Check that theme is part of the controls and that the changes in the component reflects the design in Figma. 

**Screenshots**  
-

**Additional context**  
-
